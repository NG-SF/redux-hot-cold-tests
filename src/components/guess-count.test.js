import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {GuessCount} from './guess-count';

describe('<GuessCount />', () => {
    it('Renders without crashing', () => {
        shallow(<GuessCount />);
    });

    it('Should match its snapshot', () => {
    const tree = renderer.create(<GuessCount />).toJSON();
    expect(tree).toMatchSnapshot();
    });

    it('Renders correct information', () => {
        let wrapper = shallow(<GuessCount guessCount={5} />);
        expect(wrapper.text()).toEqual("You have made 5 guesses!");
    });
});