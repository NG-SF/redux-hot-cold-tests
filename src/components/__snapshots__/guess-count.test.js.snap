// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuessCount /> Should match its snapshot 1`] = `
<h2
  id="guessCount"
>
  You have made 
  <span
    id="count"
  />
   
  guesses
  !
</h2>
`;
